data/remnant ships.txt

```diff
@@ -24,8 +24,9 @@ ship "Starling"
 		"drag" 3.8
 		"heat dissipation" 0.7
 		"fuel capacity" 900
+		"ramscoop" 1
```

Comment:
This additional ramscoop ability should be removed, as it runs counter to the idea expressed in the Catalytic Ramscoop mission that Remnant ships struggle to travel large distances; if they have built in and outfit ramscoops, then that idea is less believable. The Emergency Ramscoop outfit description should then be reverted accordingly.

Reply:
While it isn't clear to <first><last>, (and explicitly so - the Remnant have only just met the player and are unlikely to divulge their plans), when Tali says "but our ships' limited ranges struggle to take us farther" she is talking about hindering their ship's ability to sustain hard engine burns on the time scale necessary for mundane interstellar travel. Which is years and possibly decades. And face the added problem of being in interstellar space away from stars, so their effectiveness is going to be severely reduced. 

The Remnant are well aware of the fact that they are in a dead-end with nowhere to run using conventional technology. Not even with Jump Drives (and they obviously do have access to them as salvage). There is no tech within all space known to them or the player that can get the Remnant farther South or East, so they are falling back to the stand-by: Develop ships capable of interstellar travel the old-fashioned way.

While we don't see them having near-FTL drives at this point (and may well never, since the actual mechanics of the game aren't built to handle near-FTL speeds or interstellar travel), it is going to be a plot point that they are working on said drives. For lack of a good alternative, it'd basically be like planning on flying for years under constant afterburners. And even with their built in ramscoops, that'll still burn through all their fuel in seconds.

Reply:
>she is talking about hindering their ship's ability to sustain hard engine burns on the time scale necessary for mundane interstellar travel.

Well then how does that relate at all to the conversation in that mission, which relates to the ramscoop?

>There is no tech within all space known to them or the player that can get the Remnant farther South or East,

That implies that there will never be any systems to the south or east that can be jumped to. I don't think that the lack of eastern systems in the galaxy should be treated like a plot point when it will eventually be filled and therefore no longer make any sense to treat as a plot point.

>For lack of a good alternative, it'd basically be like planning on flying for years under constant afterburners. And even with their built in ramscoops, that'll still burn through all their fuel in seconds.

Well if that's the case, then I don't think any ramscoop is going to help them given the distance between stars. The amount of ramscoop ability required to be able to keep an afterburner running in interstellar space would be unrealistically immense.

Reply:
It is a matter of scale. Ditch the idea of having "a little ramscoop tucked on the bottom of the ship" and ramp up the Remnants built-in scoops into the "The main defining characteristic shaping this ship is that it is a giant ramscoop" type scale. Effectively, looking at a 100+ ton ramscoop sort of thing, with all the efficiencies that go with it. This is all theoretical, but it is the sort of thing the Remnant are looking at. And they'd want the best possible ramscoop tech to base that on. 

Well, the fact that there is no systems within JD range now kind of *is* a plot point. Whether they are out of range or simply hidden somehow, the point is they aren't there, and the Remnant most certainly *would* be there if it was there and within JD range. I don't consider this a problem though, it is simply fodder for story and missions. It is simply an opportunity.

Reply:
>Well, the fact that there is no systems within JD range now kind of is a plot point. Whether they are out of range or simply hidden somehow, the point is they aren't there,

They aren't there now, but what happens when someone does add a system there? Are you going to remove all the missions you made assuming there were no systems? Or are you preemptively preventing any systems from being made there in the first place due to the plot of the missions? This is either going to be overly restrictive on future development or internally inconsistent if systems are to be made.

Reply:
And seeing as how whatever is added to the east must be reachable through some means, it really makes no sense to try and use sub-light speed travel to reach another system when you could instead spend a fraction of the time simply finding a way around the galaxy to get there. Even if it means traveling all the way to the north of the galaxy in order to find a path eastward that then leads back down to a point of being directly east of the Waste, it'd be quicker than sub-light travel.

Reply:
OK, I don't want to block off a chunk of the galaxy to always be empty, so I'll abandon that potential plot point. 

That being said, why should the Remnant be focused on systems? If one wants to hide in a universe full of defined points, the best way to hide is to ensure that one is *between* the points in an area that no-one has the ability to conveniently reach? Systems are mainly valued as convenient concentrations of resources. If they can figure out ways to be fully self-sustaining with only minimal energy inputs (such as gathering gases from nebula and other galactic phenomena), then they can effectively create a place to hide that is out of reach of everyone.

Also keep in mind that they probably are not going to be rational at the meta level. They preferred the idea of possibly wandering to their deaths through a wild wormhole leading to a wasteland of a Nebula over the concept of being ruled by the Alphas; then they stayed there after the war (and I think there's a mention somewhere that they do, in fact, know that the war ended). 

And they have already demonstrated very effective resource management. The fact that they can arrived as refugees and now can sustain themselves and thrive on three not-terribly-hospitable planets, as well as maintain an effective defence against a full Tier 2+ race... Seriously, they must have taken recycling, efficiency, and resource collection into an art form. So for such a group, the idea of simply abandoning systems entirely to live as untraceable collections of ships in the void might be a fairly attractive idea. I'm not saying that they are going to *achieve* this anytime soon, but it makes a good goal.

Reply:
At the end of the day, though, this is getting wildly far afield of the actual issue. At the heart of it, there is no reason why the Remnant *wouldn't* want better ramscoop technology, and lots of reasons they would want it. At the end of the day, they want *more* range. The player can assume what they will, and I'm sure some of the more thoughtful ones will venture the thought "Hmmm, they've got decent fuel, and built in ramscoops. I wonder why they want *more* fuel?" 

And I'm quite happy with whatever thoughts or ideas they think up for that.

Reply:
Whatever the case may be, I don't think they need build in ramscoops, as with the most recent change the Remnant ships would now have built in shield regen, hull repair, ramscoop, AND scanners.

Reply:
Apparently so.

If I recall correctly, *every* ship has built in ramscoop, just at a very low rate. The Remnant, for a variety of reasons, have simply put a greater emphasis on this than most other shipyards.

1) They are a flight-oriented culture. They exist because they ran; and the only reason they are *still* here is because they ran out of distance to run. Regardless of how they choose to run, fuel and an independence from fueling infrastructure is critical to their ability to run.

2) Their ships are semi-organic and incorporate a much higher degree of complexity into the base hull than traditional shipyards. It fits this general complexity.

3) Their ships generally have comparatively large wingspans filled with things that look like they are designed for collecting stuff. Thus this fits their appearance.

4) While it isn't readily apparent from the current missions, it is going to become a fairly important plot point that the Remnant have broken off from the "standard" tech tree. They might steal from it, but they fundamentally aren't following the same "rules" of tech development of pug-influenced space. From a meta perspective, this is intended to add variety. We already have tons of ships, and honestly there's not much point in having a parallel series of warships to the Wanderers unless they are fundamentally and significantly different. And a significant element of that is that they are going to deviate to an increasingly large degree from the path established by the Human, Coalition/Heliarch, and Wanderer paths. (aka all the cultures that are either obviously or suspected of being influenced by the Pug)

Reply:
>just at a very low rate

0.5 or 1 is a low rate, not the 2 or 3 that the Gull, Albatross, and Pelican have. A single Wanderer ramscoop is typically all I use for my ships, and that’s 4 ramscoop. 3 and even 2 is close enough to that point where I ask why the Remnant even have a ramscoop outfit if most of their ramscoop needs are dealt with by the built in ramscoop. 

Speaking of the Emergency Ramscoop, the Remnant seem to have built in shield regen, hull repair, and now scanning because they don’t have outfits that would otherwise do that. (For the latter, ignoring the generic outfit scanner they have.) The same would not be true for ramscoop; they’d have built in ramscoop and an outfit for it too.

Reply:
Basically the Remnant do not have major fuel production facilities, and a) they operate in an area where there are no other major fuel suppliers, and b) they operate under the premises that sudden evacuation and flight are possibilities. In-flight fuel production of their ships is a standard means of refueling for them. 

That being said, I reduced the gull to 1, and the Pelican & Albatross down to 2.

Reply:
Alright, I'll take it, but I don't think any Remnant ship going into the future should have more than 2 built in.